client/web/web.go

```diff
@@ -426,18 +423,31 @@ func (s *Server) serveAPIAuth(w http.ResponseWriter, r *http.Request) {
 		default:
 			// no additional auth for this distro
 		}
-	case err != nil && (errors.Is(err, errNotOwner) ||
-		errors.Is(err, errNotUsingTailscale) ||
-		errors.Is(err, errTaggedLocalSource) ||
-		errors.Is(err, errTaggedRemoteSource)):
-		// These cases are all restricted to the readonly view.
-		// No auth action to take.
+	case err != nil && errors.Is(err, errNotOwner):
+		// Restricted to the readonly view, no
+		// auth action to take.
+		s.lc.IncrementCounter(r.Context(), "web_client_viewing_not_owner", 1)
+		resp.AuthNeeded = ""
```

Comment:
I'm not really sure why we set this to an empty string, since that's already the zero value, and it's not being set anywhere above.  Maybe this was just making that explicit or something?  @soniaappasamy do you recall?

Reply:
Ah good point. Can remove this if we didn't end up having a reason for it.